### DISCUSSION TITLE
Password Logging into Nightscout webpage for first time

### DISCUSSION THREAD

**Discussion by chhallva** [NightscoutFoundation/xDrip#2591] [Q&A] [answered]
What is the username to use when logging into Nightscout for the first time through web browser.  It asks for a username and password. 
I found my password (API Token) through Free DNS website.  The following picture is what I see when logging in through the browser on my phone and computer.  It is asking for a username and password.  I assume the password is my API Token, but I've tried the username from my freeDNS and also my email address.  After pressing enter, it just shows me the screen (same as below) to enter my username and password again.  The wording on the Xdrip site makes it sound like it would not ask for a username so I'm wondering if I set something up wrong.
![Nightscout initial set-up](https://user-images.githubusercontent.com/43630526/210158381-4c57e520-756e-4b99-ade2-8bea631b1de5.jpg)

The following picture is from this page:  https://navid200.github.io/xDrip/docs/Nightscout/NS_setup.html
![Xdrip Nigthscout setup](https://user-images.githubusercontent.com/43630526/210157079-c54465fa-a5de-469d-b5cc-9d75d4a8be00.GIF)

**Comment by Navid200** [comment, marked as answer]
Please note that you have relvealed your hostname in the screenshot you have posted.

Can you open a terminal on your virtual machine and post a screenshot of the status page?
https://navid200.github.io/xDrip/docs/Nightscout/Status.html

**Reply by chhallva** [reply]
Thanks letting me know about hostname.  Here is a screen shot of my status page.

![Status Page](https://user-images.githubusercontent.com/43630526/210158336-9d8e9e63-cdce-405d-91d5-6c868e6bbb13.GIF)

**Reply by Navid200** [reply]
Did you run installation phase 2?

After you complete installation phase 1, you will not be able to use the system until you also run phase 2.

**Reply by chhallva** [reply]
Thanks for your help!  I had run it because I remember filling in all those questions that came up.  I went ahead and reran it and now have a valid status.  Afterwards, I went to a web browser and was able to log into the nightscout site.  I left the username blank and entered my API_token which it accepted.

**Reply by Navid200** [reply]
Fantastic.
I am pleased.
Thanks for the feedback
